test/membrane_rtmp_plugin/rtmp_source_bin_test.exs

```diff
@@ -232,6 +263,17 @@ defmodule Membrane.RTMP.SourceBin.IntegrationTest do
       |> add_file_option(option_f("flv"))
       |> add_file_option(option_vcodec("copy"))
       |> add_file_option(option_acodec("copy"))
+      |> then(fn command ->
+        if ts_offset = Keyword.get(opts, :ts_offset) do
+          add_file_option(command, %FFmpex.Option{
+            name: "-output_ts_offset",
+            argument: ts_offset,
+            contexts: [:output]
+          })
+        else
+          command
+        end
+      end)
```

Comment:
I kind of don't like the way this command is conditionally added, but I don't have a particular idea how to make it simpler either.

Reply:
I will try to change it